lib/common/services/redis/templates/redis-master-statefulset.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       release: {{ .Release.Name }}
       role: master
   serviceName: {{ template "redis.fullname" . }}-headless
+  replicas: 1
```

Comment:
why don't you use a variable there?

Reply:
why so ? you can't have more than one in any case. That's why they didn't bother to specify it in the first place